__tests__/redux/reducers/createArticleReduder.test.js

```diff
@@ -0,0 +1,91 @@
+import getArticleReducer from '../../../src/redux/reducers/articles.reducer';
+import {
+  UPLOAD_IMAGE,
+  UPDATE_ARTICLE,
+  RESET_PROPS,
+  GET_MY_ARTICLES,
+} from '../../../src/redux/actions/types/article.type';
+import { article } from '../../../__mocks__/data';
+import { getFromEditor } from '../../../src/utils/getArticleItemData';
+import resetPasswordReducer from '../../../src/redux/reducers/resetPassword.reducer';
+import applyPasswordReducer from '../../../src/redux/reducers/applyPassword.reducer';
+import {
+  POST_RESET,
+  PATCH_RESET,
+} from '../../../src/redux/actions/types/resetPassword.type';
+// import Articleitem from '../../../src/components/items/Articleitem';
+
+describe('Article reduccer', () => {
+  const initialState1 = {
+    articles: [],
+    currentArticle: {},
+    uploadedImage: {},
+    updatedArticle: {},
+  };
+  it('Should return an object', () => {
+    const articles = getArticleReducer(initialState1, {
+      type: UPLOAD_IMAGE,
+      payload: {},
+    });
+    expect(typeof articles).toBe('object');
```

Comment:
@rukundoeric not only you test the datatypes but you also need to test if the state contains the data you expect.